distribution/tools/plugin-cli/build.gradle

```diff
@@ -39,3 +39,7 @@ test {
   // TODO: find a way to add permissions for the tests in this module
   systemProperty 'tests.security.manager', 'false'
 }
+
+if (project.inFipsJvm) {
+  thirdPartyAudit.enabled = false
```

Comment:
I think this definitely warrants a comment as to why we disable third party audit

Reply:
I don't understand why we are completely disabling this as opposed to excluding the forbidden classes when using a FIPS JVM?

Reply:
@jasontedor we could exclude but it's a _really_ long list, and those classes are AFAIK the only difference between fips and java 8 from a third party audit perspective, if we except all I don't think there's any reason to keep it enabled.